src/mmw/mmw/settings/layer_settings.py

```diff
@@ -259,22 +259,6 @@
             'legend_unit_breaks': [0, '', 20, '', 40, '', 60, '', 80, '', 100],
             'big_cz': False,
         },
-        {
-            'display': 'DRB 2100 Urban Centers-NP FX',
```

Comment:
Correct. This is stated (but not expounded upon) in #3328. Basically they wanted the NP layers for a specific audience, but worried that it could confuse the larger audience, so weren't sure if they were going to keep it. In one of our recent update calls, it seemed like the specific audience can use the Drexel API directly, and thus does not need this included in MMW.